TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -203,6 +203,15 @@ def makeTreeFromMiniAOD(self,process):
         self.VectorRecoCand.append("genTops(GenTops)")
         self.VarsDouble.append("genTops:weight(GenTopWeight)")
 
+        #Event weight to fix FastSim pre-UL bug
+        if self.fastsim:
+            process.FastSimWeightPR31285To36122 = cms.EDProducer("FastSimWeightPR31285To36122",
+                genCollection = cms.InputTag("prunedGenParticles"),
+                genJetTag = cms.InputTag('slimmedGenJets'),
+                recJetTag = cms.InputTag("slimmedJets")
+            )
+            self.VarsDouble.append("FastSimWeightPR31285To36122(FastSimWeightPR31285To36122)")
```

Comment:
if the branch name is the same as the producer name, it doesn't need to be repeated, i.e. `self.VarsDouble.append("FastSimWeightPR31285To36122")`

Reply:
done